src/encord_active/lib/metrics/example.py

```diff
@@ -28,6 +28,24 @@ class ExampleMetric(Metric):
 $$h_{\lambda}(x) = \frac{1}{x^\intercal x}$$
 """
 
+    def __init__(self):
```

Comment:
seems like there is a duplicated code section just above this __init__ function regarding title, metrci_type, ... Do you intend to remove that and move everything inside __init__ or else?

Reply:
Yep! done